
try{
    throw { msg:"找不到资源",code:404 }
}catch(err){
    console.log(err)
}




